pycalphad/variables.py

```diff
@@ -634,7 +789,7 @@ class ChemicalPotential(StateVariable):
     display_name = property(lambda self: f'Chemical Potential {self.species}')
 
     def __init__(self, species):
-        species = Species(species)
+        species = Component(species)
```

Comment:
We're leaving this as 'species' for the moment but will fix it in a future revision?

Reply:
I think so. I'm open to changing it